src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -849,7 +848,7 @@ public abstract partial class WebRequestPSCmdlet : PSCmdlet
         #region Abstract Methods
 
         /// <summary>
-        /// Read the supplied WebResponse object and push the
+        /// Read the supplied WebResponse object and push the.
```

Comment:
Remove the added period, it breaks the sentence.

Reply:
Fixed.